doc/api/crypto.md

```diff
@@ -99,7 +99,7 @@ changes:
   includes a public key and a challenge.
 
 ```mjs
-const { Certificate } = await import('crypto');
+import { Certificate } from 'crypto';
```

Comment:
Note this was intentionally not using static import: https://github.com/nodejs/node/pull/37594#issuecomment-794021447. I think you should remove these changes from this PR, and open a new PR to discuss those if you think it should be changed.

Reply:
Thanks for linking this to the original motivation @aduh95, I've reverted these. FWIW I disagree with the logic as there isn't a clear recovery path here. I would personally rather suggest always making the crypto module available, but providing a stub in non crypto builds that throws when attempting to use crypto to achieve the same effect. Agreed it's out of scope for this PR though.